docs/architecture/solana/migration/wallet-user.mdx

```diff
@@ -38,13 +37,14 @@ import { MigrateWallet } from '@theme/MigrateWallet'
 
 The Helium Hotspot App is no longer used to manage Tokens or perform Wallet activities and instead
 is only used to manage Original Helium Hotspots and FreedomFi Hotspots.
+Hotspots are now managed with Maker Apps or the Helium Wallet.
```

Comment:
Should downplay maker apps as we're working to bring these features back to Helium Wallet.